### DISCUSSION TITLE
How to transfer DB data to an SQL based backend system

### DISCUSSION THREAD

**Discussion by devmuaz** [pocketbase/pocketbase#4122] [Q&A]
Hi, I have been using the Pocketbase for a while now and it sounds awesome, but I was thinking if there's any way that I can transfer the data accordingly to another custom backend with an SQL database like a Laravel based app?

How to achieve such thing in the Pocketbase if such feature already exists?

**Comment by jonshipman** [comment]
Not within Pocketbase, but the database is just a standard sqlite database. Should be about 1,000 or so tools to take a sqlite database and convert it to .sql import statements for mysql.

Bear in mind that you won't be using Pocketbase anymore.

**Comment by devmuaz** [comment]
Hi @jonshipman
Sorry for being too late to reply to you, and yes I know that actually but I was wondering if such feature is considered in the future to be added to the Pocketbase which make things much easier.

Thanks mate, much appreciated!
